libs/ebpfnetsh/util.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation
```

Comment:
Rename util.h and util.cpp to avoid abbreviations.  e.g. "utilities.h" and "utilities.cpp"